core/modules/system/css/system.theme.css

```diff
@@ -240,6 +240,7 @@ abbr.ajax-changed {
 .js[dir="rtl"] fieldset.collapsed .fieldset-title {
   background-image: url(../../../misc/menu-collapsed-rtl.png);
   background-position: 98% 50%;
+  background-repeat: no-repeat;
```

Comment:
the background-repeat rule should be by default, in the upper selector

Reply:
the upper selector is for `background-image` not `background`.